src/test/java/com/softserve/edu/greencity/api/tests/ownsecurity/OwnSecurityTestRunner.java

```diff
@@ -0,0 +1,126 @@
+package com.softserve.edu.greencity.api.tests.ownsecurity;
+
+import com.softserve.edu.greencity.api.assertions.ArrayAssertion;
+import com.softserve.edu.greencity.api.builders.ownsecurity.ChangePasswordBuilder;
+import com.softserve.edu.greencity.api.builders.ownsecurity.SignInBuilder;
+import com.softserve.edu.greencity.api.builders.ownsecurity.SignUpBuilder;
+import com.softserve.edu.greencity.api.builders.ownsecurity.UpdatePasswordBuilder;
+import com.softserve.edu.greencity.api.clients.OwnSecurityClient;
+import com.softserve.edu.greencity.api.models.ownsecurity.*;
+import com.softserve.edu.greencity.api.tests.GreenCityAPITestRunner;
+import com.softserve.edu.greencity.ui.api.mail.GoogleMailAPI;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.softserve.edu.greencity.ui.tests.signin.SignInTexts.FORGOT_PASS_MAIL_SUBJECT;
+
+public class OwnSecurityTestRunner extends GreenCityAPITestRunner {
+
+        static protected OwnSecurityClient ownSecurityClient;
+        static public Response signInResponse;
+
+
+    static public ArrayAssertion getSignInAssertion(String email, String password) {
+        SignInDto credentials = SignInBuilder.signInWith()
+                .email(email)
+                .password(password)
+                .build();
+        OwnSecurityClient ownSecurityClient = new OwnSecurityClient("JSON");
+        Response response = ownSecurityClient.signIn(credentials);
+        signInResponse = response;
+        response.print();
```

Comment:
Prints to console? Anyway, you will not need it as I've made Rest-assured to log all requests and responses